### DISCUSSION TITLE
How to block the "revert" action in PullRequests

### DISCUSSION THREAD

**Discussion by rafaelwalter** [community/community#30859] [Pull Requests] [answered]
Hi,

Every time my team has a pull request with a merge commit GitHub shows a "Revert" button and it scares me that someone could click that button.

Is there a way to prevent revert in my repository or to block that button?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there!

The revert button is a UI helper to do a git operation that any user with write access to the repository can do - push a commit with changes and open a PR. This button does not automatically undo the pull request, it creates a new pull request that would revert the previous pull request IF approved and merged. More info [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/reverting-a-pull-request). 

I hope this helps assuage some of your fears 😄 .
